javascript-modal/main.js

```diff
@@ -0,0 +1,11 @@
+var $button = document.querySelector('#button');
+var $modal = document.querySelector('#container');
+var $noButton = document.querySelector('#no-button');
+
+$button.addEventListener('click', function (event) {
+  $modal.style.display = 'block';
+});
+
+$noButton.addEventListener('click', function (event) {
+  $modal.style.display = 'none';
+});
```

Comment:
This is inline styling, which is _really_ bad news. Instead, you should make a class that you can either add or remove to hide/show the element.